projects/ama-sdk/src/lib/api-implementations/acm-api/project-api.ts

```diff
@@ -85,14 +85,15 @@ export class ACMProjectApi implements ProjectApi {
         );
     }
 
-    public getAll(): Observable<Project[]> {
+    public getAll(pagination: Partial<Pagination> = {}): Observable<{ entries: Project[]; pagination: Pagination }> {
```

Comment:
For the return value please create a separate type: `PaginatedEntries<Project>`. This will be the same for every GET requests like this.

```
interface PaginatedEntries<T> {
    entries: T[];
    pagination: Pagination;
}
```